[bitnami/postgresql-ha] repmgr password disclosure in logfile

**Which chart**:
`bitnami/postgresql-ha:6.0.0`

**Describe the bug**
The repmgr password is disclosed in plaintext in the postgresql pod logs if a node goes down.

    [2020-12-08 15:14:52] [WARNING] unable to ping "user=repmgr password=<plaintext_password> host=postgresql-ha-postgresql-0.postgresql-ha-postgresql-headless.my-dev.svc.cluster.local dbname=repmgr port=5432 connect_timeout=5"
    [2020-12-08 15:14:52] [DETAIL] PQping() returned "PQPING_NO_RESPONSE"
    [2020-12-08 15:14:52] [WARNING] connection to node "postgresql-ha-postgresql-0" (ID: 1000) lost
    [2020-12-08 15:14:52] [DETAIL] 
    
    [2020-12-08 15:14:57] [ERROR] connection to database failed
    [2020-12-08 15:14:57] [DETAIL] 
    timeout expired
    
    [2020-12-08 15:14:57] [DETAIL] attempted to connect using:
    user=repmgr password=<plaintext_password> connect_timeout=5 dbname=repmgr host=postgresql-ha-postgresql-0.postgresql-ha-postgresql-headless.my-dev.svc.cluster.local port=5432 fallback_application_name=repmgr options=-csearch_path=
    [2020-12-08 15:14:57] [WARNING] reconnection to node "postgresql-ha-postgresql-0" (ID: 1000) failed
    [2020-12-08 15:14:57] [WARNING] unable to connect to local node
    [2020-12-08 15:14:57] [NOTICE] node 1000 has recovered, reconnecting
    [2020-12-08 15:14:57] [NOTICE] reconnected to local node after 0 seconds
    

**To Reproduce**
Steps to reproduce the behavior:

1. Install the helm chart with `repmgrPassword` set in the `values.yaml`
2. Delete/bounce one of the postgresql pods, e.g. `postgresql-ha-postgresql-0`
3. Tail the log of the other postgresql replica.

**Expected behavior**
The repmgr password is not disclosed in the logs of the pods on a connection/ping failure.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.4.1", GitCommit:"c4e74854886b2efe3321e185578e6db9be0a6e29", GitTreeState:"clean", GoVersion:"go1.14.11"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.11", GitCommit:"d94a81c724ea8e1ccc9002d89b7fe81d58f89ede", GitTreeState:"clean", BuildDate:"2020-03-12T21:08:59Z", GoVersion:"go1.12.17", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.6", GitCommit:"7015f71e75f670eb9e7ebd4b5749639d42e20079", GitTreeState:"clean", BuildDate:"2019-11-13T11:11:50Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
My idea on how to solve this issue was to not include the repmgr password in the conninfo string in the `repmgr.conf` and use either the ENV variable or password file option to provide the repmgr password as outlined in https://repmgr.org/docs/current/configuration-password-management.html.

I tried multiple ways to remove the password from the connection string (e.g. via a custom `files/repmgr.conf`), but wasn't able to get it working while properly replacing the environment variables in my custom conf file.

All the other properties as set in [librepmgr.sh ](https://github.com/bitnami/bitnami-docker-postgresql-repmgr/blob/master/11/debian-10/rootfs/opt/bitnami/scripts/librepmgr.sh#L398) are fine, I just would like to know a way on how to not disclose the repmgr password in the log file and hope one of you can help me out.

--- a/bitnami/postgresql-ha/Chart.yaml
+++ b/bitnami/postgresql-ha/Chart.yaml
@@ -27,4 +27,4 @@ name: postgresql-ha
 sources:
   - https://github.com/bitnami/bitnami-docker-postgresql
   - https://www.postgresql.org/
-version: 6.5.2
+version: 6.5.3

--- a/bitnami/postgresql-ha/templates/postgresql/statefulset.yaml
+++ b/bitnami/postgresql-ha/templates/postgresql/statefulset.yaml
@@ -280,8 +280,8 @@ spec:
           livenessProbe:
             exec:
               command:
-                - sh
-                - -c
+                - bash
+                - -ec
                 - '{{ include "postgresql-ha.pgpassword" . }} psql -w -U {{ include "postgresql-ha.postgresqlUsername" . | quote }} -d {{ (include "postgresql-ha.postgresqlDatabase" .) | quote }}  -h 127.0.0.1 -c "SELECT 1"'
             initialDelaySeconds: {{ .Values.postgresql.livenessProbe.initialDelaySeconds }}
             periodSeconds: {{ .Values.postgresql.livenessProbe.periodSeconds }}
@@ -295,8 +295,8 @@ spec:
           readinessProbe:
             exec:
               command:
-                - sh
-                - -c
+                - bash
+                - -ec
                 - '{{ include "postgresql-ha.pgpassword" . }} psql -w -U {{ include "postgresql-ha.postgresqlUsername" . | quote }} -d {{ (include "postgresql-ha.postgresqlDatabase" .) | quote }}  -h 127.0.0.1 -c "SELECT 1"'
             initialDelaySeconds: {{ .Values.postgresql.readinessProbe.initialDelaySeconds }}
             periodSeconds: {{ .Values.postgresql.readinessProbe.periodSeconds }}

